src/encoded/tests/features/cart.feature

```diff
@@ -8,7 +8,7 @@ Feature: Cart
         When I press "/experiments/ENCSR611ZAL/"
         And I press "/experiments/ENCSR000INT/"
         And I press "/experiments/ENCSR000DZQ/"
-        And I wait for the content to load
+        And I wait for 5 seconds
```

Comment:
Why did you use 5 seconds rather than wait until the DOM-object is loaded?

Reply:
I found that waiting for the content to load didn’t help — the test just barged on even if the cart was still loading.